backend/README.md

```
@@ -0,0 +1,82 @@
+# FreStyle backend (Go)
+
+Spring Boot から Go (Gin + GORM) に段階的に置き換える Go 版バックエンド。
+Phase 0 では基盤のみ。Phase 1 以降で `FreStyle/` 配下の Spring Boot コントローラーを機能ごとに移植する。
+
+## ディレクトリ構造（クリーンアーキテクチャ）
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)